documentation/CLI-for-JFrog-Curation.md

```diff
@@ -0,0 +1,81 @@
+JFrog CLI : CLI for JFrog Curation
```

Comment:
This new page should added to [cli.ftmap](https://github.com/jfrog/jfrog-cli/blob/v2/documentation/cli.ftmap).